docusaurus/docs/custom-templates.md

```diff
@@ -50,6 +50,14 @@ my-app/
       index.js (or index.tsx)
 ```
 
+### Testing a template
+
+To test a template locally, pass the file path to the directory of your template source using the `file:` prefix.
+
+```sh
+yarn create react-app my-app --template file:../path/to/your/template/cra-template-[template-name]
```

Comment:
@mrmckeb thoughts on changing this file to reference `npx` instead of `yarn create` and `npm init`? Seems like we've been using that in most other places?